src/js_realm.hpp

```diff
@@ -272,6 +274,9 @@ class RealmClass : public ClassDefinition<T, SharedRealm, ObservableClass<T>> {
         {"clearTestState", wrap<clear_test_state>},
         {"copyBundledRealmFiles", wrap<copy_bundled_realm_files>},
         {"deleteFile", wrap<delete_file>},
+ #if REALM_ENABLE_SYNC
+        {"_asyncOpen", wrap<async_open_realm>},
```

Comment:
So we have `Realm.openAsync()` and `Realm._asyncOpen()`? It must be a sign that we should remove `openAsync()` very soon.